tests/test_rds.py

```diff
@@ -885,6 +885,29 @@ def test_rds_snapshot_count_filter(self):
             resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_rds_data_in_transit_encrypt(self):
```

Comment:
@sgungi-intuit should rename this to `test_rds_db_option_groups`

Reply:
@darrendao took care this one.